docker-compose.yml

```diff
@@ -296,19 +296,17 @@ services:
       - COLLECTOR_OTLP_ENABLED=true
 
   keycloak:
-    image: quay.io/keycloak/keycloak:20.0.3
```

Comment:
Based on https://quay.io/repository/keycloak/keycloak?tab=tags&tag=20.0.3 if you look at the child manifests - there is a Docker image pushed with amd64 / arm64. Docker should pull the correct image. Building an image doesn't change that logic (you use the same Dockerfile FROM as image here)

Reply:
I checked that pulling with `docker compose pull` and then checking `docker inspect quay.io/keycloak/keycloak:20.0.3` that `keycloak` architecture is `"Architecture": "arm64",` on m1 mac.